lighthouse-core/scripts/update-report-fixtures.js

```diff
@@ -37,32 +34,30 @@ async function update(artifactName) {
   await cli.runLighthouse(url, flags, budgetedConfig);
   await server.close();
 
+  let newArtifacts = assetSaver.loadArtifacts(artifactPath);
+
+  // Normalize some data so it doesn't change on every update.
```

Comment:
this file was making new artifacts, saving them with `saveArtifacts()`, loading them again to normalize churn-y `Timing` then doing it's own `fs.writeFileSync()`. Normalization should just happen before the first (and only) save, which also means the concluding newline added by `saveArtifacts()` will be in the resulting file.